pkg/execution/executor/executor.go

```diff
@@ -883,7 +883,7 @@ func (e *executor) handleGeneratorStep(ctx context.Context, gen state.GeneratorO
 		GroupID:     groupID,
 		Kind:        queue.KindEdge,
 		Identifier:  item.Identifier,
-		Attempt:     0,
```

Comment:
This is fine.  We're actually enqueueing the next step to run here, and this is the first attempt of that step - so this should be 0.